src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,115 @@
+package duke;
+
+import java.util.ArrayList;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.Path;
+
+/**
+ * Storage class that interacts with the .txt file to store and retrieve data.
+ */
+public class Storage {
+    private final String HOME = System.getProperty("user.home");
+    private final Path DIR = Paths.get(HOME, "data");
+    private final Path PATH = Paths.get(HOME, "data", "iPStore.txt");
+    private final String IO_MESSAGE = "Sorry! There was an IOException! Initialising with an empty Tasklist!";
+    private final String INVALID_MESSAGE = "Sorry! There was an error in reading your data! Initialising with a semi-complete Tasklist!";
+
+    /**
+     * Constructor for Storage class.
+     */
+    public Storage() {
+    }
+
+    /**
+     * Combines all tasks into 1 long string separated by newlines.
+     *
+     * @param items List of tasks.
+     * @return A string that can be stored in the .txt file.
+     */
+    String allTasksCombined(ArrayList<Task> items) {
```

Comment:
Perhaps can consider using a verb for the method name? (E.g. CombineAllTasks etc)